src/lib.rs

```diff
@@ -0,0 +1,75 @@
+mod message_receiver;
+mod place_runner;
+mod plugin;
+mod place;
+mod core;
+
+use std::{
+    fmt,
+    path::{Path, PathBuf},
+    error::Error,
+};
+
+use crate::{
+    core::{run_place, run_script},
+    place_runner::PlaceRunnerOptions,
+    message_receiver::RobloxMessage,
+};
+
+const DEFAULT_PORT: u16 = 54023;
+const DEFAULT_TIMEOUT: u16 = 15;
```

Comment:
Should we make sure to only have these constant at only one place? Right now there are also in `main.rs`

Reply:
Good call, I forgot that I passed those around a bit. I'll fix it.